packages/terra-functional-testing/src/services/wdio-visual-regression-service/methods/BaseCompare.js

```diff
@@ -69,25 +69,25 @@ export default class BaseCompare {
    */
   getScreenshotName(context) {
     const { name } = (context.options || {});
-    const parentName = this.createTestName(context.test.parent);
     const testName = this.createTestName(name || context.test.title);
```

Comment:
Since we are requiring the name to be provided, I think we we can remove reference to context.test.title as well

Reply:
`context.test.title` is no longer referenced: https://github.com/cerner/terra-toolkit/pull/545/commits/b85221c60247a8a943bdf916e2fb00df6cee06f7